### ISSUE TITLE
reserved identifier violation

### ISSUE THREAD

**Issue by elfring** [spacetelescope/stsci.stimage#24]
I would like to point out that identifiers like “[`_STIMAGE_ERROR_H_`](https://github.com/spacetelescope/stsci.stimage/blob/222d47ba92955975ee661061f5a4481199715531/include/lib/error.h#L37 "Update candidate")” and “[`__STIMAGE_WRAP_UTIL_H__`](https://github.com/spacetelescope/stsci.stimage/blob/222d47ba92955975ee661061f5a4481199715531/src/wrap/wrap_util.h#L37 "Another update candidate")” [do eventually not fit](https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL51-CPP.+Do+not+declare+or+define+a+reserved+identifier#DCL51CPP.Donotdeclareordefineareservedidentifier-NoncompliantCodeExample%28HeaderGuard%29 "Do not declare an identifier which is reserved for the compiler implementation.") to the expected naming convention of the C++ language standard.
Would you like to adjust your selection for unique names?

**Comment by mcara** [comment]
Sorry for the delayed response, yes, go ahead.

**Comment by mcara** [comment]
> 💭 Would you become interested to [use a development tool like “clang-tidy” for corresponding source code adjustments](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/reserved-identifier.html)?

No. Not at this moment. At a later time we will refactor C code but at this moment, if you with to fix the two identifiers that you mentioned - go ahead.
